client/packages/views/index.js

```diff
@@ -34,7 +84,12 @@ const Packages = ( props ) => {
 			<CompactCard className="settings-group-card">
 				<FormSectionHeading className="settings-group-header">{ __( 'Packaging' ) }</FormSectionHeading>
```

Comment:
If the other section is called `Prefecined packages`, I would call this one something more consistent, like `Custom packages`.